continuous_integration/environment.yml

```diff
@@ -0,0 +1,13 @@
+name: test
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - pip
+  - psutil
+  - python-lmdb
+  - pytest
+  - pytest-faulthandler
+  - pytest-repeat
+  - pytest-rerunfailures
+  - pytest-timeout
```

Comment:
Not meant as a blocking comment. I have a preference for only adding pytest plugins once they're needed. We _occasionally_ run into CI breakages when a new plugin release comes out.

Reply:
Removed faulthandler and rerunfailures.
Now that we have thread-safety tests, pytest-timeout is an important tool as there's a risk of deadlock.
I routinely use pytest-repeat during debugging - it just never stays there after the final polish.